tv_apps/smart-system/js/app_modal_dialog.js

```diff
@@ -408,15 +457,15 @@
     this.elements[type].classList.remove('visible');
   };
 
-  AppModalDialog.prototype.isFocusable = function amd_isVisible() {
-    return !!this.element && this.element.classList.contains('visible');
+  AppModalDialog.prototype.isFocusable = function amd_isFocusable() {
+    return !!this.element && this.element.classList.contains('opened');
   };
 
   AppModalDialog.prototype.focus = function amd_show() {
     // XXX: Focusing smart-button fails at the second time popup if we don't
     // postpone it. We need to find the root cause.
     setTimeout(function() {
-      document.activeElement.blur();
+      this.element.focus();
```

Comment:
I think we shouldn't focus `this.element` and then focus to other buttons. Is it required because we should focus the container while animation is running? If yes, please only focus container in this case.